app/controllers/glossary_controller.rb

```diff
@@ -0,0 +1,4 @@
+class GlossaryController < ApplicationController
+  def show
```

Comment:
I remember this conversation hehe. It was my first PR in MAS about cleanup this methods that was reverted. I still agree with @esbel but since we are friends :) { friends: :beer: }